test/common/router/router_test.cc

```diff
@@ -98,6 +99,8 @@ class RouterTestBase : public testing::Test {
 
     // Make the "system time" non-zero, because 0 is considered invalid by DateUtil.
     test_time_.setMonotonicTime(std::chrono::milliseconds(50));
+
+    EXPECT_CALL(callbacks_.dispatcher_, setTrackedObject(_)).Times(AnyNumber());
```

Comment:
Is this needed due to a strict mock? If so can you comment? If not can we remove? Same below?